packages/@aws-cdk/aws-logs/test/test.loggroup.ts

```diff
@@ -115,7 +115,7 @@ export = {
     const stack2 = new Stack();
 
     // WHEN
-    const imported = LogGroup.fromLogGroupArn(stack2, 'lg', 'arn:aws:logs:us-east-1:123456789012:log-group:my-log-group');
+    const imported = LogGroup.fromLogGroupArn(stack2, 'lg', 'arn:aws:logs:us-east-1:123456789012:log-group:my-log-group:*');
```

Comment:
Please don't modify existing tests, just add a new one. 

Additionally, this test is now asserting something that is not entirely right. The `:*` at the end is not part of the ARN of the log group. I am okay with optimistically recognizing this situation and Doing The Right Thing, but passing a proper LogGroup ARN should not all of a sudden become illegal.